### ISSUE TITLE
Conflict with ultralytics and cv2.imshow

### ISSUE THREAD

**Issue by guiaugustoga987** [ultralytics/ultralytics#3005]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Detection

### Bug

Hi,

I'm trying a simple code, in which I load a model, made a detection and I need to exhibit the segmentation mask with cv2.imshow funcion. But I can't see the output with cv2.imshow function. If I run the detection without cv2.imshow it works as intended. Output : 

```
0: 640x448 1 TL, 109.2ms
Speed: 4.4ms preprocess, 109.2ms inference, 2.4ms postprocess per image at shape (1, 3, 640, 640)
```

Also, if I use CLI without `show=True` like this :   `yolo mode=predict model='/path/to/model.pt' source ='path/to/img.jpg'` I get the following output : 

```
Ultralytics YOLOv8.0.113 🚀 Python-3.8.10 torch-2.1.0a0+git86c7652 CUDA:0 (NVIDIA GeForce 940MX, 2003MiB)
YOLOv8s-seg summary (fused): 195 layers, 11779987 parameters, 0 gradients, 42.4 GFLOPs

image 1/1 /home/ga/Documents/5.jpg: 640x448 1 TL, 108.1ms
Speed: 3.2ms preprocess, 108.1ms inference, 2.8ms postprocess per image at shape (1, 3, 640, 640)
Results saved to runs/segment/predict
```
But if I use `show=True` argument there is no output in my terminal. I believe there is some sort of conflict with ultralytics and cv2.imshow because if I can't even display a simple image if I import YOLO as the code below : 

```
import cv2
from ultralytics import YOLO

img = cv2.imread('/path/to/image.jpg')

cv2.imshow('a',img)
cv2.waitKey(0) 
```

If I remove `from ultralytics import YOLO` it works as intended.

### Environment

YOLO : 8.0.113
OS : Ubuntu 20.04
Python : 3.8.10

### Minimal Reproducible Example

```
import cv2
from ultralytics import YOLO
import numpy as np

model = YOLO("/path/to/model.pt")
img = cv2.imread('path/to/image.jpg')


cv2.imshow('a',img)
cv2.waitKey(0)  

result = model(img, verbose=True)

if result[0].cpu().masks is None :
    output = np.zeros((480, 640, 3), dtype="uint8")
else :
    output = (result[0].cpu().masks.data[0].numpy() * 255).astype("uint8")

cv2.imshow('b',output)
cv2.waitKey(0)
```

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@guiaugustoga987 I'm not able to reproduce any issue with `yolo predict show`, everything works as expected for me:

https://github.com/ultralytics/ultralytics/assets/26833433/1c397e9f-c611-4d0f-9d8b-a9e15393b28b

**Comment by glenn-jocher** [comment]
The code below also works correctly for me, showing first an annotated image and then the original image:

```python
from ultralytics import YOLO
import numpy as np
model = YOLO()
img = cv2.imread('ultralytics/assets/bus.jpg')
cv2.imshow('a',img)
cv2.waitKey(0)  
result = model(img, verbose=True)
if result[0].cpu().masks is None :
    output = np.zeros((480, 640, 3), dtype="uint8")
else :
    output = (result[0].cpu().masks.data[0].numpy() * 255).astype("uint8")
cv2.imshow('b',output)
cv2.waitKey(0)
```

**Comment by guiaugustoga987** [comment]
It is strange because I can't display any image if I import `from ultralytics import YOLO`. I have a desktop with the the same OS, python and YOLO version and everyting runs smoothly.

**Comment by Laughing-q** [comment]
@guiaugustoga987 actually I encountered the same issue with you before😂. But I found it's nothing about `ultralytics` it's about `torchvision`. I can't `imshow` after `import torchvision`. You encountered this with `from ultralytics import YOLO` is also because we import torchvision internally in our code. 
This could be some environment conflicts but I never located it since I have no clue how it happens, I fixed it by creating another conda environment.

**Comment by Laughing-q** [comment]
@guiaugustoga987 If you finally locate which package have conflict with `torchvision`, please let me know! Thanks!

**Comment by guiaugustoga987** [comment]
@Laughing-q You were right. This issue has nothing to do with `ultralytics`, but with `torchvision`. There seems to have some sort of conflict between `PyAV` and `OpenCV`which leads to cv2.imshow to freeze as commented [here](https://github.com/PyAV-Org/PyAV/issues/978) I removed `PyAV` and everything runs as it should. Thank you for pointing this out. I was about to reinstall my ubuntu OS to solve it!

**Comment by Laughing-q** [comment]
@guiaugustoga987 It's good to know that you've solved the issue! I'm going to close the issue, please feel free to reopen it if you have related issues! Thanks!
